config/events.yaml

```
@@ -0,0 +1,7 @@
+sylius_ui:
+    events:
+        sylius.admin.configurable_product.show.variant_content:
+            blocks:
+                pricing:
+                    template: '@SyliusPriceHistoryPlugin/Product/Show/_variantContentPricing.html.twig'
```

```suggestion
                    template: '@SyliusPriceHistoryPlugin/Admin/Product/Show/_variantContentPricing.html.twig'
```
I'd be for introducing a top-level split between admin and shop in templates to reduce cognitive load and the chance of mixing contexts

---

now valid 😅